src/main/java/hudson/plugins/git/extensions/impl/PreBuildMerge.java

```diff
@@ -58,7 +58,7 @@ public Revision decorateRevisionToBuild(GitSCM scm, Run<?, ?> build, GitClient g
             return rev;
 
         // Only merge if there's a branch to merge that isn't us..
-        listener.getLogger().println("Merging " + rev + " onto " + remoteBranchRef + " using " + scm.getUserMergeOptions().getMergeStrategy().toString() + " strategy");
+        listener.getLogger().println("Merging " + rev + " onto " + remoteBranchRef + " using " + scm.getUserMergeOptions().getMergeStrategy().toString() + " strategy and "+scm.getUserMergeOptions().getFastForwardMode().toString());
```

Comment:
`Merging: {}, to: {}, strategy: {}, fastword: {}. `
even:
`Merging: {}, to: {}, options: {} `
where for options is called `scm.getUserMergeOptions.toString()`